src/awkward/operations/ak_from_buffers.py

```diff
@@ -182,12 +196,15 @@ def _reconstitute(form, length, container, getkey, backend, byteorder, simplify)
 
     elif isinstance(form, ak.forms.BitMaskedForm):
         raw_array = container[getkey(form, "mask")]
-        excess_length = int(math.ceil(length / 8.0))
+        if length is unknown_length:
+            next_length = unknown_length
+        else:
+            next_length = int(math.ceil(length / 8.0))
```

Comment:
"Excess length" is terminology that I even copied over [into Julia](https://github.com/jpivarski/AwkwardArray.jl/blob/4f97922ded63937d45f4b4c3dd474f379d95fd3f/src/AwkwardArray.jl#L3435). But `next_length` is fine.

The idea here is that bit masks can only come in units of bytes, so $\frac{7}{8}$ of the time, you'll have more in your mask than you should. That's the sense in which it's an excess.